### PR TITLE
chore: schema change for web sdk support

### PR DESCRIPTION
**Pull request by amliuyong**
----

*By submitting this pull request, I confirm that my contribution is made under the terms of the Apache-2.0 license*

## Summary

add new field: `device.host_name`

(describe what this merge request does)

## Implementation highlights

(describe how the merge request does for feature changes, share the RFC link if it has)

## Test checklist

- [ ] add new test cases
- [ ] all code changes are covered by unit tests
- [ ] end-to-end tests
  - [ ] deploy control plane with CloudFront + S3 + API gateway
  - [ ] deploy control plane within VPC
  - [ ] deploy ingestion server
    - [ ] with MSK sink
    - [ ] with KDS sink
    - [ ] with S3 sink
  - [x] deploy data processing
  - [ ] deploy data modeling
    - [ ] new Redshift Serverless
    - [ ] provisioned Redshift
    - [ ] Athena
  - [ ] deploy with reporting

## Is it a breaking change

- [ ] add parameters without default value in stack
- [ ] introduce new service permission in stack
- [ ] introduce new top level stack module

### CODE REVIEW

File: src/data-pipeline/spark-etl/gradle.properties [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Project
 group=software.aws.solution
-projectVersion=1.0.0
+projectVersion=1.0.1
```
Comment (zxkane) [inline src/data-pipeline/spark-etl/gradle.properties:3]: the next release in main is `1.1.0`

File: src/data-pipeline/spark-etl/src/main/resources/data_schema.json [hunk 2]
```diff
@@ -151,6 +151,12 @@
       "nullable": true,
       "metadata": {}
     },
+    {
```
Comment (zxkane) [inline src/data-pipeline/spark-etl/src/main/resources/data_schema.json:154]: Did you test upgrading from v1.0.0 to latest version with the schema changes?

**Review by zxkane — CHANGES_REQUESTED** [review]

File: src/data-pipeline/spark-etl/gradle.properties (hunk 1 above)
Comment (amliuyong) [reply src/data-pipeline/spark-etl/gradle.properties]: changed.

File: src/data-pipeline/spark-etl/src/main/resources/data_schema.json (hunk 2 above)
Comment (amliuyong) [reply src/data-pipeline/spark-etl/src/main/resources/data_schema.json:154]: yes. test upgroup from v1.0.0, no issue, event with existing files, the new added filed will be `null` for existing data.

**Review by zxkane — APPROVED** [review]

**Comment by zxkane** [comment]
Let's cherry-pick it to v1.0.x patch branch with corresponding version.

**Pull request merged**